callstats-java-sdk/src/main/java/io/callstats/sdk/httpclient/CallStatsHttpClient.java

```diff
@@ -1,390 +0,0 @@
-package io.callstats.sdk.httpclient;
-
-import io.callstats.sdk.internal.CallStatsConst;
-import io.callstats.sdk.internal.CallStatsUrls;
-import io.callstats.sdk.internal.listeners.CallStatsHttpResponseListener;
-
-import java.io.FileInputStream;
-import java.io.FileNotFoundException;
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.UnsupportedEncodingException;
-import java.net.URI;
-import java.net.URISyntaxException;
-import java.util.List;
-import java.util.Properties;
-
-import org.apache.commons.lang3.StringUtils;
-import org.apache.http.HttpResponse;
-import org.apache.http.NameValuePair;
-import org.apache.http.client.ClientProtocolException;
-import org.apache.http.client.HttpClient;
-import org.apache.http.client.entity.UrlEncodedFormEntity;
-import org.apache.http.client.methods.HttpPost;
-import org.apache.http.client.methods.HttpUriRequest;
-import org.apache.http.concurrent.FutureCallback;
-import org.apache.http.entity.StringEntity;
-import org.apache.http.impl.client.HttpClients;
-import org.apache.http.impl.conn.PoolingHttpClientConnectionManager;
-import org.apache.http.impl.nio.client.CloseableHttpAsyncClient;
-import org.apache.http.impl.nio.client.HttpAsyncClients;
-import org.apache.http.impl.nio.conn.PoolingNHttpClientConnectionManager;
-import org.apache.http.impl.nio.reactor.DefaultConnectingIOReactor;
-import org.apache.http.impl.nio.reactor.IOReactorConfig;
-import org.apache.http.message.BasicHeader;
-import org.apache.http.nio.client.HttpAsyncClient;
-import org.apache.http.nio.reactor.ConnectingIOReactor;
-import org.apache.http.nio.reactor.IOReactorException;
-import org.apache.logging.log4j.LogManager;
-import org.apache.logging.log4j.Logger;
-
-/**
- * The Class CallStatsHttpClient.
- *
- * @author Karthik Budigere
- */
-public class CallStatsHttpClient {	
```

Comment:
Do you know if this used by anyone? It's always a bit risky to remove public classes.

Reply:
No one can use this directly, its not exposed. Its internal use only.